integration_tests/raw/api_test.go

```diff
@@ -515,14 +536,84 @@ func (s *apiTestSuite) TestEmptyValue() {
 	s.Equal([]byte{}, oldVal)
 }
 
+func (s *apiTestSuite) storeAddr(id uint64) string {
+	return fmt.Sprintf("store%d", id)
+}
+
+type storeSafeTsMockClient struct {
+	tikv.Client
+	requestCount int32
+}
+
+func (c *storeSafeTsMockClient) SendRequest(ctx context.Context, addr string, req *tikvrpc.Request, timeout time.Duration) (*tikvrpc.Response, error) {
+	if req.Type != tikvrpc.CmdStoreSafeTS {
+		return c.Client.SendRequest(ctx, addr, req, timeout)
+	}
+	atomic.AddInt32(&c.requestCount, 1)
+	resp := &tikvrpc.Response{}
+	resp.Resp = &kvrpcpb.StoreSafeTSResponse{SafeTs: 150}
+	return resp, nil
+}
+
+func (c *storeSafeTsMockClient) Close() error {
+	return c.Client.Close()
+}
+
+func (c *storeSafeTsMockClient) CloseAddr(addr string) error {
+	return c.Client.CloseAddr(addr)
+}
+
+func (s *apiTestSuite) TestGetStoreMinResolvedTS() {
```

Comment:
I don't think it's proper to put `resolved_ts` test inside `integration_tests/raw/api_test.go` since it's for the RawKV API testing.

Reply:
move to `pd_api_test` file